components/org.wso2.carbon.identity.api.server.input.validation/org.wso2.carbon.identity.api.server.input.validation.v1/src/main/java/org/wso2/carbon/identity/api/server/input/validation/v1/core/ValidationRulesManagementApiService.java

```diff
@@ -107,6 +126,33 @@ public List<ValidationConfigModel> updateInputValidationConfiguration(
         }
     }
 
+    /**
+     * Method to update input validation configuration.
+     *
+     * @param validationConfigModel Validation Configuration Model.
+     * @param tenantDomain          Tenant domain name.
+     * @return ValidationConfigModel for the field.
+     */
+    public ValidationConfigModel updateInputValidationConfigurationForField(
+            ValidationConfigModel validationConfigModel, String tenantDomain) {
+
+        try {
+            isFieldFound(validationConfigModel.getField());
+            List<ValidationConfigModel> configModels = new ArrayList<>();
+            configModels.add(validationConfigModel);
+            List<ValidationConfiguration> requestDTO = buildRequestDTOFromValidationRequest(configModels);
+            validateProperties(requestDTO, tenantDomain);
+            ValidationConfiguration configurations = InputValidationServiceHolder.getInputValidationMgtService()
+                    .updateValidationConfiguration(requestDTO.get(0), tenantDomain);
+            return buildResponse(configurations);
+        } catch (InputValidationMgtException e) {
+            if (LOGGER.isDebugEnabled()) {
```

Comment:
I think this debug log is redundant.

Reply:
Addressed with commit 0c6e61b04b60aeaf760e03c611b6a26105759859